src/client-functions/selector-builder/add-api.js

```diff
@@ -87,8 +89,20 @@ async function getSnapshot (getSelector, callsite) {
     return node;
 }
 
-function addSnapshotPropertyShorthands (obj, getSelector) {
-    SNAPSHOT_PROPERTIES.forEach(prop => {
+function assertSnapshotExtensionOptions (extensions) {
+    if (!isNullOrUndefined(extensions)) {
+        assertObject('extendSnapshot', '"extendSnapshot" option', extensions);
+
+        Object.keys(extensions).forEach(prop => {
+            assertFunction('extendSnapshot', `extendSnapshot '${prop.replace(/'/g, "\\'")}'`, extensions[prop]);
```

Comment:
Why we need to escape `'` in property name here? Also I guess it's better to change text to `Snapshot extensions method` or smthg like that